packages/controls/src/OrbitControl.ts

```diff
@@ -29,7 +29,10 @@ export class OrbitControl extends Script {
   zoomSpeed: number;
   enablePan: boolean;
   autoRotate: boolean;
-  autoRotateSpeed: number;
+  /**
+   * The radian of automatic rotation per second.
+   */
+  autoRotationAngle: number = Math.PI;
```

Comment:
I think  autoRotationAngle not expression  rotation  per second, I feel  you only rotation `PI` once，then  stop. very strange.
I think autoRotateSpeed is better.